assets/scss/components/compat/edd/_generic.scss

```diff
@@ -0,0 +1,22 @@
+.edd_purchase_submit_wrapper .edd-cart-added-alert {
+	display: block;
+}
+
+.edd_purchase_submit_wrapper .edd-icon.edd-icon-check {
+	vertical-align: middle;
+	width: 15px;
+}
+
+.edd_download_purchase_form .edd_price_options {
+	margin-bottom: 10px;
+	/* stylelint-disable-next-line */
+	& ul{
```

Comment:
I think the `&` here is unnecessary. Is there a reason that I might oversee to add it? 👀